sections/introduction.md

```diff
@@ -0,0 +1,49 @@
+# Introduction
+
+> ℹ️ These docs are written for engineers and ad ops professionals looking to integrate zero.js in their site.
+
+## Background
+
+The programmatic advertising system emits worrying amounts of CO2. In the worst cases, delivering a single impression involves the processing, storing and transmitting of data across hundreds of nodes. The result of this noisy system is that a single impression can emit over 1 gram of carbon dioxide 🤯
+
+> ℹ️ Programmatic advertising falls into an Advertiser's *scope 3* (supply chain) emissions and so they must reduce these emissions to meet net zero targets (offsetting is insufficient).
+
+The Glimpse Zero system enables publishers to sell their inventory in a carbon efficient manner. This inventory is called a Green Media Product. In the next section we'll provide an overview for how we do this.
+
+## Ad decisioning process
+
+The ad server decides which creative to serve after all bids are received and a call is made to the publisher's ad server. We call this a *postmatch*. A consequence of postmatching is that when certain types of campaign run (like Sponsorships) the ad server discards all client-side bids to (rightly) prioritises the Sponsorship deal.
+
+![Ad decision postmatch](assets/introduction-ad-postmatch.png)
+*Fig 1. An overview of a postmatch ad decision process*
+
+To illustrate how Glimpse Zero differs we enable a client-side *prematch* process. For example, consider a Sponsorship line item with the following configuration:
+
+- runs on Wednesdays for 4 weeks
+- runs from 0800 - 1800
+- targets devices in a specific region
+- only delivers once per user per day
+- targeting 100% of eligible impressions
+
+Our client-side library, zero.js, brings targeting criteria into the browser to prematch ad units and divide them into matched and unmatched groups when it makes sense to do so. Unmatched units are passed through to the publisher's existing header bidding calls while matched units are removed.
```

Comment:
should we use a different phrase than 'header bidding calls' as it could be TAM etc? Would 'ad calls' work / 'auctions'?

Reply:
i did consider exactly this and ended up on header bidding 😂 I'm thinking of header bidding as "code that runs in the browser to solicit bids before calling the ad server" and so i was including TAM/UAM in that mix.